common/entryCreditBlock/ecblockBody.go

```diff
@@ -11,55 +11,64 @@ import (
 	"github.com/FactomProject/factomd/common/primitives"
 )
 
+// ECBlockBody contains all the entry information for the EC block. The entries will be one of 5 types:
+// CommitChain, CommitEntry, IncreaseBalance, MinuteNumber, or ServerIndexNumber
```

Comment:
link to https://github.com/FactomProject/FactomDocs/blob/master/factomDataStructureDetails.md#ecid-bytes